src/Subsampling/include/gudhi/sparsify_point_set.h

```diff
@@ -11,6 +11,8 @@
 #ifndef SPARSIFY_POINT_SET_H_
 #define SPARSIFY_POINT_SET_H_
 
+#include <boost/iterator/function_output_iterator.hpp>
```

Comment:
It seems this has been introduced in Boost 1.66 and gudhi requires Boost ≥ 1.56.0.
This modification requires some documentation change.

Reply:
It was `boost/function_output_iterator.hpp` before that, I should probably `#if` the boost version unless we consider that 1.66 is recent enough that we can require it (end of 2017, that may be a bit too recent).